k2/csrc/array_ops.h

```diff
@@ -172,12 +173,14 @@ void ExclusiveSumDeref(Array1<T *> &src, Array1<T> *dest);
                        summation is over row axis (slower because we have to
                        transpose), axis = 1 means summation is over column axis.
  */
+
 template <typename T>
-void ExclusiveSum(ContextPtr &c, Array2<T> &src, Array2<T> *dest, int axis);
+void ExclusiveSum(ContextPtr &c,
```

Comment:
implementation should be in arra_ops_inl.cuh

Reply:
done